### PR TITLE
05_28_2022/gcbui/create_character

### PR DESCRIPTION
**Pull request by gcbui**
First branch for game

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -23,24 +23,32 @@ def init_background(self, background_symbol, width, height):
...
       
 current_frame = Frame()
+ 
+def draw(x,y,symbol=None,symbols=None):
+    if symbols == None:
+        i = int((height-y) * (width) + (x-1))
+        current_frame.image = current_frame.image[:i] + symbol + current_frame.image[i + 1:]
+    else:
+        for symbol in symbols:
+            i = int((height-y-symbol[1]) * (width) + (x+symbol[0]-1))
```
Comment (BolajiMakinde) [inline main.py]: Indexing is off by 1 on x and y placements

File: main.py [hunk 2]
```diff
@@ -23,24 +23,32 @@ def init_background(self, background_symbol, width, height):
...
+            current_frame.image = current_frame.image[:i] + symbols[symbol] + current_frame.image[i + 1:]
 
-def draw(x,y,symbol):
-    i = int((height-y) * (width) + (x-1))
-    current_frame.image = current_frame.image[:i] + symbol + current_frame.image[i + 1:]
-
-    
 def render_frame():
     print(current_frame.banner+current_frame.image)
```
Comment (BolajiMakinde) [inline main.py:38]: remove extra spacing

File: main.py [hunk 3]
```diff
@@ -23,24 +23,32 @@ def init_background(self, background_symbol, width, height):
...
 
 
 def game():
+    character_hash_map = {tuple([2,1]):'|', tuple([8,1]):'|', tuple([3,2]):'_',tuple([4,2]):'_',tuple([5,2]):'|',tuple([6,2]):'_',tuple([7,2]):'_', tuple([1,3]):'|',tuple([5,3]):'|',tuple([9,3]):'|',tuple([2,4]):'_', tuple([3,4]): '_',tuple([4,4]):'_',tuple([5,4]):'|',tuple([6,4]):'_',tuple([7,4]):'_',tuple([8,4]):'_', tuple([5,5]):'>'                        
+,tuple([4,6]):'0',tuple([6,6]):'0',tuple([4,7]):'~',tuple([6,7]):'~'}
+
```
Comment (BolajiMakinde) [inline main.py:42]: remove extra space

**Review by BolajiMakinde — CHANGES_REQUESTED** [review]

File: main.py (hunk 3 above)
Comment (gcbui) [reply main.py:42]: removed spacing

File: main.py (hunk 2 above)
Comment (gcbui) [reply main.py:38]: removed spacing

File: main.py (hunk 1 above)
Comment (gcbui) [reply main.py]: corrected indexing

**Review by BolajiMakinde — APPROVED** [review]
looks good

**Pull request merged**